crates/service/src/service.rs

```diff
@@ -135,13 +136,30 @@ pub async fn run() -> anyhow::Result<()> {
         let ipfs_fetcher: Arc<dyn IpfsFetcher> =
             Arc::new(IpfsClient::new("https://api.thegraph.com/ipfs/").unwrap());
 
+        // TODO: Try to re-use the same watcher for both DIPS and TAP
+        let watcher = escrow_accounts_v2(
```

Comment:
worth checking with the Semiotic team but I'm 99% sure we need escrow_accounts_v1, as v2 is the one for after we launch Horizon

Reply:
That's correct.